CHANGELOG.md

```diff
@@ -52,6 +52,7 @@
 - Minor: Removed timestamp from AutoMod messages. (#3503)
 - Minor: Added ability to copy message ID with `Shift + Right Click`. (#3481)
 - Minor: Colorize the entire split header when focused. (#3379)
+- Minor: Added margin to emote search. (#3543)
```

Comment:
I believe this can just be slapped into `Minor: Add search to emote popup. (#3404, #3527)`
`(#3404, #3527, #3543)`